source/MaterialXRender/Handlers/TestObjLoader.cpp

```diff
@@ -121,6 +124,17 @@ bool TestObjLoader::load(const std::string& fileName, MeshList& meshList)
                 dump << "vn " << val1 << " " << val2 << " " << val3 << std::endl;
             }
         }
+        else if (_readGroups && line.substr(0, 2) == "g ")
```

Comment:
Detect "g <name>" which is a mesh sub-group and create partitions as the tinobjloader does.